### DISCUSSION TITLE
Docker port already in use

### DISCUSSION THREAD

**Discussion by CloudCoder** [mudler/LocalAI#3416] [Q&A] [answered]
Hi, I'm trying to install LocalAI on a vps that already has Nextcloud installed.

I cannot instal LocalAI because the port is already in use.

Any help greatly appreciated.

**LocalAI version:**
latest

**Environment, CPU architecture, OS, and Version:**
x86 ubuntu 20.04 8GB ram, 80GB ssd

**Describe the bug**
Bind for 0.0.0.0:8080 failed:

Status: Downloaded newer image for localai/localai:latest-aio-cpu
docker: Error response from daemon: driver failed programming external connectivity on endpoint local-ai (f1e729150af9d46ade84eb538f23246c8fa26dc8c7e8d66303d20b743911abba): Bind for 0.0.0.0:8080 failed: port is already allocated.

**To Reproduce**
run:
docker run -p 8080:8080 --name local-ai -ti localai/localai:latest-aio-cpu

**Expected behavior**
port 8080 in use should be detected and a message given with an option to change this port during installation

Thanks

**Comment by Nyralei** [comment, marked as answer]
Just change the port to 8081 or something
for your command:
docker run -p 8081:8080 --name local-ai -ti localai/localai:latest-aio-cpu

**Reply by CloudCoder** [reply]
Thank you Nyralei
